src/components/Button.js

```diff
@@ -8,7 +8,7 @@ import React from 'react';
 
 import {getBEMClassName} from 'utils';
 
-export const VARIANTS = ['', 'primary', 'secondary', 'anchor', 'danger', 'image', 'icon-only'];
+export const VARIANTS = ['', 'primary', 'secondary', 'anchor', 'danger', 'icon-only'];
```

Comment:
`image` variant was cleaned up an not used anymore as far as I can tell -> one less thing to support